### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by slecorne**
- refactor period picker
- refactor type picker
- remove unused code for history
- change history scroll to el-scrollbar

### CODE REVIEW

File: webpack/src/assets/base.css [hunk 1]
```diff
@@ -15,8 +15,19 @@
   --activeWeight: var(--normal);
 }
 
+.el-scrollbar {
+  --el-scrollbar-bg-color:#616161;
```
Comment (jamest1903) [inline webpack/src/assets/base.css:19]: Small detail but should we use the variable "dark-grey" here?
--dark-grey: #616161;

I guess its not something that would be changed if updating the theme.

**Review by jamest1903 — APPROVED** [review]

File: webpack/src/assets/base.css (hunk 1 above)
Comment (slecorne) [reply webpack/src/assets/base.css:19]: ah yes, I will change it

Comment (slecorne) [reply webpack/src/assets/base.css]: I also fixed the button stylings if you want to check

File: webpack/src/assets/base.css [hunk 2]
```diff
@@ -15,8 +15,16 @@
   --activeWeight: var(--normal);
 }
 
+.el-scrollbar {
+  --el-scrollbar-bg-color: var(--dark-grey);
+  --el-scrollbar-opacity: 1;
+  --el-scrollbar-hover-bg-color: var(--dark-grey);
+  --el-scrollbar-hover-opacity: 1
+  background-color: white;
```
Comment (jamest1903) [inline webpack/src/assets/base.css]: I do not see this white background. When i inspect the dom there is this class we can target to set it
class="el-scrollbar__bar is-vertical"

**Review by jamest1903 — APPROVED** [review]

**Pull request merged**